### PR TITLE
building-images.rst: add instructions to create .bmap file

### PR DESCRIPTION
**Pull request by gvancuts**
Add instructions on how to generate a bmap file that can be used
by the bmaptool when flashing an Ostro image.

[skip ci]

Signed-off-by: Geoffroy Van Cutsem <geoffroy.vancutsem@intel.com>

### CODE REVIEW

**Comment by kad** [comment]
better also to add same comments near setting in local.conf.sample

**Comment by gvancuts** [comment]
Good point @kad, I will do this and push a new update

**Comment by gvancuts** [comment]
Updated the PR, rebased and force-pushed.

File: doc/howtos/building-images.rst [hunk 1]
```diff
@@ -207,6 +207,13 @@ Example::
 
 will create both the raw and the VirtualBox appliance images, both compressed.
 
+To get the best benefit out of :command:`bmaptool` when creating your bootable media (see :ref:`booting-and-installation`),
```
Comment (dbkinder) [inline doc/howtos/building-images.rst]: How about:  The :command:`bmaptool` works best creating your bootable media (see :ref:`booting-and-instalation`) when the corresponding block map ``.bmap`` file is also generated for the image.  This ``.bmap`` file contains empty block and checksum information about the image that lets  :command:`bmaptool` optimize copying the image to your bootable media and verify what was copied.  To do this ...

**Comment by kad** [comment]
One more instance we can change:
In meta-ostro/classes/ostro-image.bbclass:
OSTRO_VM_IMAGE_TYPES ?= "dsk dsk.ova"
to 
OSTRO_VM_IMAGE_TYPES ?= "dsk dsk.bmap dsk.ova"

**Comment by gvancuts** [comment]
I have just addressed @dbkinder and @kad suggestions and force-pushed a new patch

**Comment by kad** [comment]
👍

**Pull request merged**